crypto/Cargo.toml

```diff
@@ -21,7 +21,8 @@ openmls_traits = "0.1"
 # FIXME: This is being pulled only because of flaky error types from openmls
 tls_codec = "0.2"
 serde_json = "1.0"
-derive_more = { version = "0.99", features = ["deref"] }
+derive_more = { version = "0.99", features = ["deref", "from", "display"] }
+paste = "1.0"
```

Comment:
Isn't `paste` a dependency of the proc_macro? We probably don't need it in the `crypto` crate